### DISCUSSION TITLE
Tested 6.0 preview 3 with the EF Core test suite - all green!

### DISCUSSION THREAD

**Discussion by ErikEJ** [dotnet/SqlClient#3051] [General]
https://github.com/dotnet/efcore/pull/35232

**Comment by cheenamalhotra** [comment]
Thanks @ErikEJ !

The 6.0-preview3 version is how exactly the stable 6.0.0 will be, it can be looked at as a 6.0 Release Candidate. While we prep for GA, let us know if you identify any issues with backwards compatibility (unwanted breaking changes), and they will be addressed on priority.

**Reply by ErikEJ** [reply]
Sounds great, the EF Core test suite is a good smoke test platform.
